src/main/webapp/app/modeling-submission/modeling-submission.component.html

```diff
@@ -8,7 +8,7 @@ <h2>
             </h2>
         </div>
         <div class="col-12 col-lg-4 text-right">
-            <div *ngIf="participation"><jhi-result [participation]="participation"></jhi-result></div>
+            <div *ngIf="participation"><jhi-updating-result [participation]="participation"></jhi-updating-result></div>
```

Comment:
The score here is properly updated when the result changes, but the assessment is not displayed/updated in the assessment section on the right of the page. To make that work we have to subscribe to result changes in the modeling submission component and update the displayed assessment accordingly. @krusche do we want to have that?

Reply:
if you want to have a static result just we can revert it simply to `<jhi-result>`

Reply:
I suggest that we send the result including the feedback from the server to the client, so that it could be displayed directly without having another REST call.

Reply:
done in https://github.com/ls1intum/ArTEMiS/pull/448